staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3367,7 +3367,6 @@ type PodAffinityTerm struct {
 	// pod labels will be ignored. The default value is empty.
 	// The same key is forbidden to exist in both matchLabelKeys and labelSelector.
 	// Also, matchLabelKeys cannot be set when labelSelector isn't set.
-	// This is an alpha field and requires enabling MatchLabelKeysInPodAffinity feature gate.
```

Comment:
This is a beta field and requires enabling MatchLabelKeysInPodAffinity feature gate (enabled by default).

Same below

Reply:
Updated.